cloud/linode/loadbalancers_test.go

```diff
@@ -17,6 +17,10 @@ import (
 	"strings"
 	"testing"
 
+	ciliumclient "github.com/cilium/cilium/pkg/k8s/client/clientset/versioned/typed/cilium.io/v2alpha1"
+	"github.com/linode/linode-cloud-controller-manager/cloud/annotations"
+	"github.com/linode/linode-cloud-controller-manager/cloud/linode/client"
+	"github.com/linode/linode-cloud-controller-manager/cloud/linode/firewall"
```

Comment:
nit: I think we want to keep the package imports for the CCM on the bottom. I think our golang-ci linter might be missing this check

Reply:
Updated.